vb-be/src/modules/auth/auth.service.ts

```diff
@@ -38,16 +38,15 @@ export class AuthService {
     );
   }
 
-  async createTokens(user: UserType): Promise<string[]> {
+  async createTokens(user: ITokenUser): Promise<string[]> {
     if (!user) {
       throw new ErrorHandler(HttpStatus.Error, ErrorText.TokenError);
     }
 
     try {
-      const { email, id } = user;
```

Comment:
`id` was coming in as `undefined` every time, due to how we originally create the tokens on login.  Removed `id` here for now, but will fix everything tomorrow afternoon

Reply:
Yep it should be `vbUser`. I'm pushing a fix

Reply:
I think you can get the `id` back in the service for the token generation. It should be fine now